optaplanner-docs/src/antora.yml

```diff
@@ -0,0 +1,3 @@
+name: optaplanner-docs
+title: OptaPlanner User Guide
+version: latest
```

Comment:
Interestingly enough, the IntelliJ IDEA is capable of rendering images in the docs only if this configuration file is present. It seems the IDEA's asciidoc plugin can detect this file and then assume the typical directory structure (by overriding the `imagesdir` attribute).

Removing unnecessary attributes from .adoc files will be a separate PR.